src/main/AndroidManifest.xml

```diff
@@ -63,20 +63,31 @@
       android:label="@string/title_metrics">
     </activity>
 
+
```

Comment:
just used the formatter provided by hawkular and copy pasted one to add only one added activity.
So indentation is that is used by hawkular only